packages/blocks/src/lib/navigation-map/components/nav/obstacles.svelte

```diff
@@ -70,10 +70,9 @@ const handleGeometryInput =
   };
 
 const handleOrientationInput =
-  (name: string, geoIndex: number) => (event: CustomEvent<number>) => {
+  (name: string, geoIndex: number) => (value: number) => {
     const index = $obstacles.findIndex((obstacle) => obstacle.name === name);
-    $obstacles[index]!.geometries[geoIndex]!.pose.orientationVector.th =
-      event.detail;
+    $obstacles[index]!.geometries[geoIndex]!.pose.orientationVector.th = value;
```

Comment:
nit: Could we avoid using the assertion operator and wrap this in an if statement?

Reply:
Yeah, there's a buuuunch of these in blocks. I haven't added any new null assertions, mind if I instead remove them in the new navigation map in app? We'll be able to delete this directory shortly after and just wholesale remove them.

Reply:
Works for me!